src/backend/joanie/tests/core/test_models_course_product_relation.py

```diff
@@ -14,7 +14,7 @@ def test_model_course_product_relation_uri(self):
         CourseProductRelation instance should have a property `uri`
         that returns the API url to get instance detail.
         """
-        relation = factories.CourseProductRelationFactory()
+        relation = factories.CourseProductRelationFactory(course__code="C_0001-2")
```

Comment:
```suggestion
        relation = factories.CourseProductRelationFactory(course__code="Cc_0001-2")
```
To try to match every characters in your regex ?

Reply:
Course code are normalized. This is already tested in course model test suite.